documentation-site/static/examples/radio/disabled.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {Radio, RadioGroup} from 'baseui/radio';
+
+export default function Disabled() {
+  return (
+    <RadioGroup name="radio group" value="1">
+      <Radio disabled value="1">
```

Comment:
adding `disabled` prop to the `RadioGroup` should make all the radios disabled

Reply:
thanks!